app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="log-container">
+    <div class="mx-auto max-w-lg text-center">
+      <h1 class="login-head">Budget App</h1>
+
+      <p class="lp">
+        Log in to your account to continue
+      </p>
+    </div>
```

Comment:
- [ ] Please update your splash screen to match the original design 👍 

- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/228996064-30befe9f-19c2-49a3-aecc-c2c898879079.png)

- Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/228996150-895d518b-40fc-45fd-ac82-648497bc70c8.png)